testing/base.go

```
@@ -217,3 +218,23 @@ func DumpTestLogsAfter(timeout time.Duration, c *gc.C, cleaner TestCleanup) {
 		close(done)
 	})
 }
+
+type PackageManagerSuite struct {
+	PackageManager    string
+	RepositoryManager string
+}
+
+func (s *PackageManagerSuite) SetUpTest(c *gc.C) {
+	os, _ := version.GetOSFromSeries(version.Current.Series)
```

Never swallow errors in tests. Assert that it's nil.